books/free-programming-books-uk.md

```diff
@@ -18,28 +19,32 @@
 
 ### Haskell
 
-* [Вивчити собі Хаскель на велике щастя!](http://haskell.trygub.com) - Міран Ліповача
+* [Вивчить собі Хаскела на велике щастя!](http://haskell.trygub.com) - Міран Ліповача
 
 
 ### Java
 
-* [Програмування на Java для дітей, батьків, дідусів та бабусь](http://myflex.org/books/java4kids/java4kids.htm) - Яків Файн
+* [Програмування мовою Java для дітей, батьків, дідусів та бабусь](http://myflex.org/books/java4kids/java4kids.htm) - Яків Файн
```

Comment:
this doesn't seem to be a ukrainian language resource

Reply:
Ok. Removed it.